### PR TITLE
styling the button

### PR DESCRIPTION
**Pull request by lillian-busingye**
styling the button

### CODE REVIEW

**Comment by lillian-busingye** [comment]
@Ann-Barnshaw WDT of my code?

**Comment by Ann-Barnshaw** [comment]
Hi @lillian-busingye,
The code generally looks good to me.
However, I think you could improve on the button styling. Maybe you could change the background color to a more eye-friendly color and improve the border design. WDYT?

**Comment by lillian-busingye** [comment]
@Ann-Barnshaw thank you for that review. let me work on it 👍

File: index.html [hunk 1]
```diff
@@ -21,7 +21,7 @@ <h2>HAPPY VALENTINES DAY</h2>
                     <img src="/image.jpg" alt="image" style="width: 100px;">
                     <p>Cheers from Lilian. B</p>
                 </div>
-                <button class="button" style="background: red; border:5px;">
+                <button class="button" style="background: lightskyblue; border-radius: 10px;">
```
Comment (Ann-Barnshaw) [inline index.html:24]: Hello this looks okay now

**Pull request merged**